src/scf-release/jobs/acceptance-tests/spec

```diff
@@ -4,7 +4,7 @@ name: acceptance-tests
 description: "The acceptance tests errand can be configured to run full acceptance test suite against a specific Cloud Foundry API endpoint. Consider running smoke tests errand for a smaller set of tests."
 
 packages:
-  - golang1.13.5
+  - golang1.10
```

Comment:
Please describe (at least in the commit message) _why_ we're reverting golang to 1.10